coordinator-tool/src/bin/election-coordinator.rs

```diff
@@ -624,6 +647,39 @@ async fn handle_decrypt(
     Ok(())
 }
 
+async fn handle_reset(
+    endpoint: sdk::Endpoint,
+    contract: ContractAddress,
+    wallet_path: Option<std::path::PathBuf>,
+    guardians: Vec<AccountAddress>,
+) -> anyhow::Result<()> {
+    let client = sdk::Client::new(endpoint.clone()).await?;
+    let mut contract_client =
+        contract_client::ContractClient::<ElectionContract>::create(client, contract).await?;
```

Comment:
Since this is a sensitive operation we should be a bit more careful. We should

- print which guardians are going to be excluded (print their URLs that are in the contract (see electionconfig)).
- **ask for confirmation after dry run but before sending the transaction** (one of the other commands is already doing that, so just check how we do it there).